Makefile

```diff
@@ -1266,10 +1266,8 @@ else
 	@echo "To install (requires internet access) run: $ make format-cpp-build"
 endif
 
-ifeq ($(V),1)
-  CPPLINT_QUIET =
-else
-  CPPLINT_QUIET = --quiet
+CPPLINT_QUIET = --quiet
+
 endif
```

Comment:
I think the `endif` needs to be removed as well.

Reply:
Oops :smile: I removed it now.